openpype/hosts/maya/plugins/publish/validate_skeleton_rig_content.py

```diff
@@ -0,0 +1,138 @@
+import pyblish.api
+from maya import cmds
+
+from openpype.pipeline.publish import (
+    PublishValidationError,
+    ValidateContentsOrder
+)
+
+
+class ValidateSkeletonRigContents(pyblish.api.InstancePlugin):
+    """Ensure skeleton rigs contains pipeline-critical content
+
+    The rigs optionally contain at least two object sets:
+        "skeletonAnim_SET" - Set of only bone hierarchies
+        "skeletonMesh_SET" - Set of all cacheable meshes
+
+    """
+
+    order = ValidateContentsOrder
+    label = "Rig Contents"
```

Comment:
Make sure to update the labels so it's clear that it's solely about the "skeleton rig" part.
Also, could we target this validators using `families` so they are only ever run when the instance is actually expected to need/have skeleton meshes?

Like in the Collect Rig For FBX plugin we e.g. append a family like `rig.fbx` or alike so that we target all of these validators to `rig.fbx` family?